src/main.rs

```diff
@@ -44,6 +44,10 @@ struct Args {
     /// Derive AvroSchema for generated record structs
     #[clap(long)]
     pub derive_schemas: bool,
+
+    /// Extract Derives for generated record structs
+    #[clap(long)]
+    pub extra_derives: Vec<String>,
```

Comment:
There is no option for `--extra_derive`. 
As I said earlier it is not really needed but I wonder how it is supposed to work now.